project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
 import sbt._
 
 object Dependencies {
-  private val akkaHttpCirceIntegrationV = "1.35.3"
-  private val akkaHttpV = "10.1.12" // scala-steward:off (CROM-6619)
-  private val akkaV = "2.5.31" // scala-steward:off (CROM-6637)
+  private val akkaHttpCirceIntegrationV = "1.37.0"
```

Comment:
I cannot find any release notes/changelog/version history saying what has changed. Going to have to rely on tests here. :(

Reply:
Here's [a TL;DR changelog](https://github.com/hseeberger/akka-http-json/compare/v1.35.3...v1.37.0)... that effectively points to other changelogs 🪀🐶